bin/setenv-docker-customize.sh

```diff
@@ -568,6 +571,10 @@ else
   add_in_exo_configuration "exo.agenda.office.connector.enabled=${EXO_AGENDA_OFFICE_CONNECTOR_ENABLED}"
   add_in_exo_configuration "exo.agenda.office.connector.key=${EXO_AGENDA_OFFICE_CONNECTOR_CLIENT_API_KEY}"
 
+  # Rememberme Token expiration
+  add_in_exo_configuration "exo.token.rememberme.expiration.value=${EXO_TOKEN_REMEMBERME_EXPIRATION_VALUE}"
```

Comment:
Please check the impact when the property is not set because an entry will be always introduced in the docker properties file